ports/ffmpeg/vcpkg.json

```
@@ -430,6 +430,12 @@
     "gpl": {
       "description": "Allow use of GPL code, the resulting libs and binaries will be under GPL"
     },
+    "harfbuzz": {
+      "description": "Needed for drawtext filter",
+      "dependencies": [
+        "harfbuzz"
+      ]
+    },
```

IMO features should describe capabilities. AFAIU the desired capability is the drawtext filter. harfbuzz is only the capability's dependency.
```suggestion
    "drawtext": {
      "description": "Enable the drawtext filter",
      "dependencies": [
        "harfbuzz"
      ]
    },
```

---

However, from the [upstream documents](https://ffmpeg.org/ffmpeg-filters.html#toc-drawtext-1), it is found that enabling drawtext requires turning on two compilation options. If we only use this one option to say that the drawtext function is enabled, is this description inaccurate?

---

This supports my point. The user wants drawtext capabilities. The port maintainer has to model the proper configuration for that feature.

Look at the existing "freetype" feature: "Needed for drawtext filter". 
Does this feature add anything useful at all if "drawtext" isn't enabled without "harfbuzz"? I don't know, but today I think it doesn't.

And there are features "fontconfig" and "fribidi" which add more capabilities to the drawtext filter.
Do these features add anything useful at all if "drawtext" isn't enabled without "harfbuzz"? I don't know, but today I think they don't.
Do these features express there relation to "drawtext"? Today they don't (and can't). I think they should have a dependency on the capability which they add value value to.

It is not uncommon for features to be named after dependencies, but it is a questionable pattern.